policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/medical_expenses/maximum_family_income.yaml

```diff
@@ -0,0 +1,66 @@
+description: Quebec limits the maximum family income corresponding to the amount of medical expenses.
+brackets:
+  - threshold:
+      2022-01-01: 1
```

Comment:
is it 0 if below 1?

Reply:
No, the threshold is medical expense (user defined), which is at least 1 CAD.

This table is family income limit corresponding to the amount of medical expenses

https://www.revenuquebec.ca/en/citizens/income-tax-return/completing-your-income-tax-return/how-to-complete-your-income-tax-return/line-by-line-help/451-to-480-refund-or-balance-due/line-462/point-1/